librz/arch/isa/mips/il/mips32.c

```diff
@@ -0,0 +1,1435 @@
+	// HW breakpoint
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_c_cond_fmt(csh *handle, cs_insn *insn) {
+	// Floating Point Compare
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_ceil_l_fmt(csh *handle, cs_insn *insn) {
+	// Fixed Point Ceiling Convert to Long Fixed Point
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_ceil_w_fmt(csh *handle, cs_insn *insn) {
+	// Floating Point Ceiling Convert to Word Fixed Point
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_cfc1(csh *handle, cs_insn *insn) {
+	// Move Control Word From Floating Point
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_cfc2(csh *handle, cs_insn *insn) {
+	// Move Control Word From Coprocessor 2
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_clo(csh *handle, cs_insn *insn) {
+	// Count Leading Ones in Word
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_clz(csh *handle, cs_insn *insn) {
+	// Count Leading Zeros in Word
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_ctc1(csh *handle, cs_insn *insn) {
+	// Move Control Word to Floating Point
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_ctc2(csh *handle, cs_insn *insn) {
+	// Move Control Word to Coprocessor 2
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_cvt_d_fmt(csh *handle, cs_insn *insn) {
+	// Floating Point Convert to Double Floating Point
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_cvt_l_fmt(csh *handle, cs_insn *insn) {
+	// Floating Point Convert to Long Fixed Point
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_cvt_ps_s(csh *handle, cs_insn *insn) {
+	// Floating Point Convert Pair to Paired Single
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_cvt_s_fmt(csh *handle, cs_insn *insn) {
+	// Floating Point Convert to Single Floating Point
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_cvt_s_pl(csh *handle, cs_insn *insn) {
+	// Floating Point Convert Pair Lower to Single Floating Point
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_cvt_s_pu(csh *handle, cs_insn *insn) {
+	// Floating Point Convert Pair Upper to Single Floating Point
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_cvt_w_fmt(csh *handle, cs_insn *insn) {
+	// Floating Point Convert to Word Fixed Point
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_deret(csh *handle, cs_insn *insn) {
+	// Debug Exception Return
+	// PC = DEPC
+	return NULL;
+}
+
+static RzILOpEffect *mips32_il_div(csh *handle, cs_insn *insn) {
+	RzILOpPure *rs = MIPS32_REG(0);
+	RzILOpPure *rt = MIPS32_REG(1);
+
+	RzILOpPure *quotient = SDIV(DUP(rs), DUP(rt));
+	RzILOpPure *remainder = SMOD(rs, rt);
+
+	RzILOpEffect *set_lo = SETG(MIPS32_REG_LO, quotient);
+	RzILOpEffect *set_hi = SETG(MIPS32_REG_HI, remainder);
+	return SEQ2(set_lo, set_hi);
+}
+
+static RzILOpEffect *mips32_il_div_fmt(csh *handle, cs_insn *insn) {
+	// Floating Point Divide
+	const char *rd = REG(0);
+	RzILOpPure *rs = MIPS32_REG(1);
+	RzILOpPure *rt = MIPS32_REG(2);
+	// TODO: rounding mode is defined in FCSR register
```

Comment:
Consider adding those `TODOs` into the PR description. So you don't forget about them here or open an issue after merge.